### PR TITLE
design changes in CLI commands

### PR DESCRIPTION
**Pull request by k-anshul**

### CODE REVIEW

**Comment by k-anshul** [comment]
cc @jkhwu @nishantmonu51

File: cli/cmd/deploy/deploy.go [hunk 1]
```diff
@@ -107,10 +111,19 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 				}
 
 				if err := auth.Login(ctx, cfg, redirectURL); err != nil {
+					if errors.Is(err, deviceauth.ErrAuthenticationTimedout) {
+						warn.Println("Rill login timed out because you did not confirm the code in the browser.")
```
Comment (k-anshul) [inline cli/cmd/deploy/deploy.go:115]: @begelundmuller Can the login prompt timeout for any other error apart from user not confirming the code ?

Comment (nishantmonu51) [reply cli/cmd/deploy/deploy.go:115]: change text to - "Rill login has timed out as the code was not confirmed in the browser."

File: cli/cmd/deploy/deploy.go [hunk 2]
```diff
@@ -219,14 +237,19 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 				Variables:        variables,
 			})
 			if err != nil {
+				if s, ok := status.FromError(err); ok && s.Code() == codes.PermissionDenied {
+					errorWriter.Printf("You do not have the permissions needed to create a project in org %q. Please reach out to your Rill admin.\n", cfg.Org)
```
Comment (nishantmonu51) [inline cli/cmd/deploy/deploy.go:241]: Nit: Nice to print username `User <username> does not have permissions`

**Review by nishantmonu51 — APPROVED** [review]
LGTM, Left minor comments.

**Pull request merged**